plugins/completion/coq-thirdparty.nix

```diff
@@ -0,0 +1,65 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.coq-thirdparty;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.coq-thirdparty = {
+    enable = mkEnableOption "coq-thirdparty";
+
+    package = helpers.mkPackageOption "coq-thirdparty" pkgs.vimPlugins.coq-thirdparty;
+
+    sources = mkOption {
+      type = types.listOf (types.submodule {
+        freeformType = types.attrsOf types.str;
```

Comment:
Wouldn't that be types.attrsOf types.anything ?

Reply:
Yes ! Fixed :)